packages/nvidia_nat_core/tests/nat/tools/test_datetime_tools.py

```
@@ -0,0 +1,64 @@
+    result = await runner.test_tool(config_type=CurrentTimeZoneToolConfig, input_data="unused input")
+    assert result is not None
+
+    result_match = re.match(expected_result_pattern, result)
+    assert result_match is not None, f"Result '{result}' does not match expected pattern: {expected_result_pattern}."
+
+    timezone_str = result_match.group(1)
+
+    # Validate that the result is a valid timezone string
+    try:
+        zoneinfo.ZoneInfo(timezone_str)
+    except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)